tests/resources/vocabularies/test_names_vocabulary.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Name resource tests."""
+
+import pytest
+from invenio_records_resources.proxies import current_service_registry
+from invenio_vocabularies.contrib.names.api import Name
+
+
+@pytest.fixture(scope="module")
+def names_service():
+    return current_service_registry.get("rdm-names")
+
+
+@pytest.fixture()
+def example_name(
+    app, db, es_clear, superuser_identity, names_service
+):
+    """Example name."""
+    data = {
+        "name": "Doe, John",
+        "given_name": "John",
+        "family_name": "Doe",
+        "identifiers": [
+            {
+                "identifier": "0000-0001-8135-3489",
+                "scheme": "orcid"
+            }, {
+                "identifier": "gnd:4079154-3",
+                "scheme": "gnd"
+            }
+        ],
+        "affiliations": [
+            {
+                "name": "CustomORG"
```

Comment:
Does not tests the inner relation link with affiliations, but:
- We are testing it in invenio-vocabularies
- The transformer does not support searching and adding `id` anyway, so all will be added as `name`.